libcaf_core/caf/detail/squashed_int.hpp

```diff
@@ -26,34 +26,28 @@
 namespace caf::detail {
 
 /// Compile-time list of integer types types.
-using int_types_by_size =
-  detail::type_list<                      // bytes
-    void,                                 // 0
-    detail::type_pair<int8_t, uint8_t>,   // 1
-    detail::type_pair<int16_t, uint16_t>, // 2
-    void,                                 // 3
-    detail::type_pair<int32_t, uint32_t>, // 4
-    void,                                 // 5
-    void,                                 // 6
-    void,                                 // 7
-    detail::type_pair<int64_t, uint64_t>  // 8
+using int_types_by_size = detail::type_list< // bytes
+  void,                                      // 0
+  detail::type_pair<int8_t, uint8_t>,        // 1
+  detail::type_pair<int16_t, uint16_t>,      // 2
+  void,                                      // 3
+  detail::type_pair<int32_t, uint32_t>,      // 4
+  void,                                      // 5
+  void,                                      // 6
+  void,                                      // 7
+  detail::type_pair<int64_t, uint64_t>       // 8
   >;
 
 /// Squashes integer types into [u]int_[8|16|32|64]_t equivalents
 template <class T>
 struct squashed_int {
   using tpair = typename detail::tl_at<int_types_by_size, sizeof(T)>::type;
-  using type = 
-    typename std::conditional<
-      std::is_signed<T>::value,
-      typename tpair::first,
-      typename tpair::second
-    >::type;
+  using type = std::conditional_t<std::is_signed<T>::value, //
+                                  typename tpair::first,    //
```

Comment:
No, this is on purpose. I sometimes use comments as "protected line breaks" if `clang-format` otherwise butchers an expression.

It's a bit hacky, but the alternative would be to use `// clang-format (on|off)`, which adds a lot more noise to the code and completely disables `clang-format` (which means adding an outer scope requires manual indentation, etc.). Maybe there are some tweaks to the `.clang-format` file that improves formatting of such statements reasonably well, but so far I had no luck.

Also, the current `clang-format` release does not understand the C++17-style `..._v<T>` variables and occasionally freaks out completely if something doesn't fit into a single line.